mne/preprocessing/ica.py

```diff
@@ -811,7 +822,7 @@ def _export_info(self, info, container, add_channels):
         ch_names = []
         ch_info = info['chs'] = []
         for ii in range(self.n_components_):
-            this_source = 'ICA %03d' % (ii + 1)
+            this_source = 'ICA %03d' % (ii)
```

Comment:
I'd rather do `[0]` than `#1`. But earlier you [seemed happy with the current implementation](https://github.com/mne-tools/mne-python/pull/4320#issuecomment-308731710)...